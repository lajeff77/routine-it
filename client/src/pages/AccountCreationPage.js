import React from 'react';
import MainHeader from '../components/MainHeader';

const AccountCreationPage = () => {

    return (
        <div>
            <MainHeader />
        </div>
    )
}

export default AccountCreationPage;