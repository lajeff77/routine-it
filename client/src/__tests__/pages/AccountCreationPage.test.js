import { render} from "@testing-library/react";
import React from "react";
import { MemoryRouter, Route, Routes } from "react-router-dom";
import AccountCreationPage from '../../pages/AccountCreationPage';



describe(AccountCreationPage.name, () => { 
    test('AccountCreationPage matches snapshot', () => {
        const { asFragment } = render(
            <MemoryRouter>
            <Routes>
                <Route element={<AccountCreationPage />} path="/signup" />
            </Routes>
            </MemoryRouter>
        );
    
        expect(asFragment()).toMatchSnapshot();
    });
});