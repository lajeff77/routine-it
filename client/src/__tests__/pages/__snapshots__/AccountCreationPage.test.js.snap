// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountCreationPage AccountCreationPage matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      id="main-header"
    >
      <a
        class="header-logo"
        href="/"
      >
        <h1>
          Routine It
        </h1>
      </a>
      <a
        class="header-link"
        href="/"
      >
        <p>
          Home
        </p>
      </a>
      <a
        class="header-link"
        href="/signup"
      >
        <p>
          Sign Up
        </p>
      </a>
      <a
        class="header-link"
        href="/signin"
      >
        <p>
          Sign In
        </p>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
